src/main/java/com/google/sps/servlets/PopulateUserEntity.java

```diff
@@ -32,15 +32,26 @@
 import com.google.gson.Gson;
 
 @WebServlet("/user")
-public class popUserEntity extends HttpServlet {
+public class PopulateUserEntity extends HttpServlet {
 DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
+     /**
+    * Populates the datastore with "user" entities. 
+    * Every time someone logs in, this doPost is called
```

Comment:
"Every time someone logs in" -- You may decide to change this in the future, so I recommend not writing it down here, or else this comment will go out of date.

In general, I try to avoid describing the caller's behavior when documenting the callee. Sometimes it's useful to bend the rule by documenting *hypothetical* or *suggested* caller behavior. E.g. "For example, you could safely call this whenever a user logs in." Or *required* caller behavior, e.g. "This must be called at least once for every new user."